Services/Notifications/classes/class.ilNotificationConfig.php

```diff
@@ -398,3 +390,42 @@ public function getLanguageModule()
         return $this->language_module;
     }
 }
+
+class ilNotificationLink
+{
+    private $title;
+    private string $url;
+
+    public function __construct($title, string $url)
+    {
+        $this->title = $title;
+        $this->url = $url;
+    }
+
+    public function getTitle()
+    {
+        return $this->title;
+    }
+
+    public function setTitle($title) : void
+    {
+        $this->title = $title;
+    }
+
+    /**
```

Comment:
Please remove all obsolete(!) PHPDoc comments which do not provide any added value.